test/src/actions/ios/welcome.actions.js

```diff
@@ -25,6 +26,34 @@ class WelcomeActions extends AndroidWelcomeActions {
         }
         FilesHelper.editJsonByKey(walletData, "masterKey", masterKey);
     }
+
+    async addMasterKey() {
+        for (let i = 0; i < 24;) {
+            for (let j = 1; j <= 3; j++) {
+                await ActionHelper.setText(Pages.welcomePage.typeNemonic(j), importData.importWallet[i]);
+                await ActionHelper.pause(500);
+                if (await $(Pages.welcomePage.wordSuggestion).isDisplayed()) {
+                    await ActionHelper.click(Pages.welcomePage.wordSuggestion);
+                }
+                i++
+            }
+            await ActionHelper.click(Pages.welcomePage.rightNavigationArrow);
+        }
+    }
+
+    async setPin() {
+        await ActionHelper.pause(6000);
+        for (let i = 0; i <= 3; i++) {
+            await ActionHelper.click(Pages.welcomePage.pin(importData.pass[i]));
+        }
+    }
+
+    async home() {
+        await expect(await $(Pages.welcomePage.landing).isDisplayed())
```

Comment:
this will be defined in the framework guidelines, but in general, we should not have 1 liner method.
Additionally, verifications (expect method), should & could go directly on the test layer (steps)